lib/redis_memo/memoizable/bump_version.lua

```diff
@@ -0,0 +1,34 @@
+--[[
```

Comment:
Nice! Thanks for adding the summary.

I think it's also worth mention it only works as expected after Redis 6 (prior Redis versions have a Lua OOM bug)